src/components/Navbar/Navbar.tsx

```diff
@@ -10,21 +13,39 @@ import { MobileMenu } from './MobileMenu';
 
 type NavbarProps = BoxProps;
 
-export const MENU_ITEMS = [
-  {
-    name: 'Docs',
-    href: 'https://docs.keypom.xyz',
-  },
-  {
-    name: 'Get in touch',
-    href: 'https://twitter.com/keypomxyz',
-  },
-];
+export const FIRST_LOGGED_IN_KEY = 'just_logged_in';
 
 export const Navbar = (props: NavbarProps) => {
+  const navigate = useNavigate();
   const { isLoggedIn } = useAuthWalletContext();
+
+  useEffect(() => {
+    if (isLoggedIn && !(get(FIRST_LOGGED_IN_KEY) === 'true')) {
```

Comment:
i think we should move this logic to AuthWalletContext because this logic would be coupled with Navbar

Reply:
ideally the wallet should return or redirect to another url, tried this here in walletSelector.ts
```
setupNearWallet({ successUrl: MY_DROPS_URL }),
setupMyNearWallet({ successUrl: MY_DROPS_URL }),
```
but the protected route prevents us from accessing that page and session storage got set later on.

Reply:
i can't use `const navigate = useNavigate()` in auth wallet context though 🤔  i think navbar is a good component to use this since it's practically part of the layout.